lib/result-view.coffee

```diff
@@ -108,36 +108,39 @@ class ResultView
             console.log "ResultView: Rendering as MIME", mimeType
             console.log "ResultView: Rendering as ", htmlElement
 
-            if mimeType is 'text/plain'
-                @element.classList.remove 'rich'
+            previousText = @getAllText()
+            container.appendChild htmlElement
 
-                previousText = @getAllText()
-                text = result.data['text/plain']
+            if mimeType is 'text/plain'
+                text = @getAllText()
                 if previousText is '' and text.length < 50 and
                 text.indexOf('\n') is -1
-                    @setMultiline false
-
-                    @tooltips.add atom.tooltips.add container,
-                        title: "Copy to clipboard"
-
-                    container.onclick = =>
-                        atom.clipboard.write @getAllText()
-                        atom.notifications.addSuccess "Copied to clipboard"
+                    if @element.classList.contains('rich') is false
+                        @setMultiline false
+
+                    try
+                        pre = container.getElementsByTagName('pre')[0]
```

Comment:
For example `matplotlib` shows your plot like this if `plt.show()`isn't used:
<img width="565" alt="bildschirmfoto 2016-04-01 um 18 23 46" src="https://cloud.githubusercontent.com/assets/13285808/14213232/2ae55920-f837-11e5-9d77-c410f0857f27.png">

With `container` the image would be clickable as well despite just copying the text.